Monika After Story/game/script-topics.rpy

```diff
@@ -4703,6 +4703,71 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['Psychological Quiz']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('Monika_Psychological_Quiz')
+    monika_random_topics.append('Monika_Psychological_Quiz')
+
+label Monika_Psychological_Quiz:
+    python:
+        cat_number = none
```

Comment:
In python, case is important. If you want to leverage the `None` type, then use `None`. However, since this is a number, **I recommend you set this to 0 here**